### PR TITLE
Temp. Workaround broken CentOS/EPEL pylint package

### PR DESCRIPTION
**Pull request by cevich**
Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

File: .papr.yml [hunk 1]
```diff
@@ -55,6 +55,8 @@ env:
 # present, tests will run after a successful build. Full
 # UTF-8 values are supported.
 tests:
+    - yum install -y --skip-broken pylint
+    - yum downgrade -y pylint
```
Comment (edsantiago) [inline .papr.yml]: This is the sort of thing where I like to add big screaming comments explaining what's going on, why, and when it should be time to revisit / review / remove this. Likewise for the commented-out pylint above. Aside from that, LGTM but we'll have to see if it works.

Comment (cevich) [inline .papr.yml]: Yes, this is just a 'see if it works' thing.  Skookum comments (with spell-checking) can come later 😁

**Comment by cevich** [comment]
This is the thing that breaks:

``--> Processing Dependency: python-astroid >= 1.4.5 for package: python2-pylint-1.6.5-4.el7.noarch``

**Comment by cevich** [comment]
Okay, @edsantiago I ``--force`` pushed with better comments, and it's working.  Though...uggg I never spell checked 😖

**Pull request merged**